src/Symfony/Component/Routing/RouteCollectionBuilder.php

```diff
@@ -192,12 +175,9 @@ public function setDefault($key, $value)
      * Sets a requirement that will be added to all embedded routes (unless that
      * requirement is already set).
      *
-     * @param string $key
-     * @param mixed  $regex
```

Comment:
The value passed as `$regex` is ultimately passed to `Route::setRequirement()` which has always performed a type check for `string`. Passing something other than a string here will result in an exception.